gradoop-checkstyle/src/main/resources/gradoop/checkstyle-suppressions.xml

```diff
@@ -20,6 +20,12 @@
   <suppress checks="LineLength"
             files="SocialNetworkGraph.java"
             lines="0-68"/>
+  <suppress checks="IllegalInstantiation"
+            files="PropertyValueTest.java"
+            lines="650-850"/>
+  <suppress checks="IllegalInstantiation"
+            files="PropertyEncoder.java"
+            lines="45-55"/>
```

Comment:
Is it really necessary to suppress the check on multiple lines here? If not, i think we should suppress checks in the most limited way possible in non-test files.

Reply:
It is not, but lines often change a little bit with minor Javadoc or style changes.

Reply:
ok makes sense